src/main/java/com/commercetools/project/sync/SyncerFactory.java

```diff
@@ -92,29 +93,31 @@ public static SyncerFactory of(
                 new CompletableFuture[0]))
         .thenCompose(
             ignored -> {
-              final List<CompletableFuture<Void>>
-                  categoryAndInventoryAndCartDiscountAndCustomerSync =
-                      asList(
-                          CategorySyncer.of(sourceClient, targetClient, clock)
-                              .sync(runnerNameOptionValue, isFullSync)
-                              .toCompletableFuture(),
-                          CartDiscountSyncer.of(sourceClient, targetClient, clock)
-                              .sync(runnerNameOptionValue, isFullSync)
-                              .toCompletableFuture(),
-                          InventoryEntrySyncer.of(sourceClient, targetClient, clock)
-                              .sync(runnerNameOptionValue, isFullSync)
-                              .toCompletableFuture(),
-                          CustomerSyncer.of(sourceClient, targetClient, clock)
-                              .sync(runnerNameOptionValue, isFullSync)
-                              .toCompletableFuture());
+              final List<CompletableFuture<Void>> categoryAndInventoryAndCartDiscountAndCustomer =
```

Comment:
Why this change?

Reply:
These changes are not related to this PR.